app/components/UI/Select.js

```diff
@@ -123,25 +159,39 @@ class Select extends React.PureComponent {
                   onFocus: openMenu,
                   onMouseDown: toggleMenu
                 })}
+                initialValue={selectedItem ? selectedItem.value : null}
+                forwardedRef={this.inputRef}
+                css={{
+                  cursor: 'pointer',
+                  color: 'transparent',
+                  'text-shadow': `0 0 0 ${color || theme.colors.primaryText}`
+                }}
               />
             </div>
             <SelectOptionList {...getMenuProps({}, { suppressRefError: true })}>
               {isOpen
                 ? items.map((item, index) => (
```

Comment:
Using bulky maps in-place tend to make `JSX` harder to read and modify. Consider extracting this map into a local function like `selectOptions`

Reply:
I've pulled this out into a separate render method.